doc/getting_started/getting_started.rst

```diff
@@ -164,24 +185,22 @@ packaged, the one you compiled on your own, or the one you downloaded from the
 net.  The Zephyr build system doesn't know about them and doesn't officially
 support them.
 
-As already noted above, the SDK also includes prebuilt host tools.  To use the
-SDK's prebuilt host tools alongside a 3rd party or custom cross-compiler, keep
-the ZEPHYR_SDK_INSTALL_DIR environment variable set to the Zephyr SDK
-installation directory. To build without the Zephyr SDK's prebuilt host tools,
-the ZEPHYR_SDK_INSTALL_DIR environment variable must be unset
+As already noted above, the SDK also includes prebuilt host tools. To
+use the SDK's prebuilt host tools alongside a 3rd party or custom
+cross-compiler, keep the ZEPHYR_SDK_INSTALL_DIR variable set to the
```

Comment:
Add the double back-ticks on the environment variable name:

    ``ZEPHYR_SDK_INSTALL_DIR``